packages/volto/src/config/Blocks.jsx

```diff
@@ -351,6 +351,12 @@ const blocksConfig = {
     restricted: false,
     mostUsed: true,
     sidebarTab: 1,
+    allowedPeertubeInstances: [
+      'freediverse.com', // About: https://freediverse.com/about/instance
+      'fair.tube', // About: https://fair.tube/about/instance
+      'tube.reseau-canope.fr', // About: https://tube.reseau-canope.fr/about/instance
+      'peertube.eus', // About: https://peertube.eus/about/instance
+    ],
```

Comment:
Do we want all of these to be enabled by default?

Reply:
Idk. I think and hope that no one of each instances goes against the CoC of the organization. Would this be enough for add them as default?

Reply:
I kind of hate to say it but I think for that reason we have to consider adding new video sources as a breaking change, and should keep the default list empty at least in Volto 18

Reply:
@ionlizarazu Thanks, this looks okay to me now.

We've disabled the HTML block for that reason.

I've worked with other systems that have a central configuration for what domains/URLs are allowed to be embedded. That could be useful for some Plone sites, and make it easier to implement consent for cookies in a more consistent way -- but it is definitely a bigger topic than we can tackle in this PR.